lib/cdo/rack/attack.rb

```diff
@@ -57,4 +58,23 @@ def write?
       req.write? && %r{^/v3/shared-properties/([^/]+)/[^/]+$}.match(req.path).try(:[], 1)
     end
   end
+
+  ActiveSupport::Notifications.subscribe('rack.attack') do |_, _, _, _, req|
+    if CDO.newrelic_logging
```

Comment:
This is explicitly configurable so that we can enable it in unit tests without necessarily enabling other things that come with setting `rack_env` to `production`.